runners/dmn-tck-reporter/src/main/resources/templates/index.ftl

```diff
@@ -196,6 +196,9 @@
                                     <dt>Failed: </dt><dd>${vendor.failed}</dd>
                                     <dt>Skipped: </dt><dd>${header.totalTests - vendor.succeeded - vendor.failed}</dd>
                                     <dt>Info: </dt><dd>${vendor.comment}</dd>
+                        		<#if vendor.tckRunnerManualUrl?has_content>
+									<dt>TCK Runner manual: </dt><dd>This vendor has supplied an instruction manual for user to reproduce TCK test results, available <a href="${vendor.tckRunnerManualUrl}" target="_blank">at this link <span class="fa fa-external-link"></span></a>.</dd>
+								</#if>
```

Comment:
Suggestion for wording: 

* if the instruction URL is available:

> Instructions: to execute the tests and reproduce the results, please follow the vendor provided instructions at this link.

* if instructions are not provided:

> Instructions: this vendor did not provide instructions to execute the tests. Please contact the vendor directly.